### ISSUE TITLE
Scheduled posts sent empty Instant Articles when published

### ISSUE THREAD

**Issue by webadictos** [Automattic/facebook-instant-articles-wp#198]
Hi,

I have an issue with scheduled posts, when they are published i got this error: 

Empty Instant Article: The Instant Article does not have any content.

I need to enter to the wordpress editor and "update" the post so it can be send correctly to Instant Article.

This happen only with scheduled posts.

Thank you,

**Comment by MLRobbins** [comment]
Also seeing this behavior. Any update?

**Comment by diplatis** [comment]
Same here.

**Comment by MattiaConfalonieri** [comment]
Same here... is ridiculous...

**Comment by Aioros** [comment]
So, I can pinpoint the reason for the empty scheduled articles here:

```
protected function _get_the_content() {

global $post, $more;

if ( ! $post ) {
	return '';
}
```
(class-instant-articles-post.php, line 291, v2.10)

The global variable `$post ` is not set up when wp-cron publishes a scheduled post. Right after this block the `$post` variable is checked and properly setup.
That return line seems to be there for some specific reason, but I don't know what it is. Hope this helps.

**Comment by emanuelet** [comment]
Still experiencing the same issue

**Comment by MattiaConfalonieri** [comment]
This is a real pain in the ass guys...

**Comment by chrisackermann** [comment]
Sorry for the headache here. We've confirmed the PR from @Aioros addresses the issue where scheduled posts are not issued to Facebook. We're performing some additional testing to ensure there are no unintended side effects and will plan to merge tomorrow if all's good.

**Comment by msnadler** [comment]
I'm likely late too the party, but I'm having a similar issue with scheduled posts. Though in my case, I don't get the error message others reference, the scheduled posts just don't show up at all.
Unscheduled posts work fine.

**Comment by chrisackermann** [comment]
We've now released Version 2.11 of the plugin which includes the fix provided in #257 from @Aioros (thanks for the contribution!).  Please upgrade to this latest version and let us know if you have any further issues with scheduled posts.
